docs/usage/overview.md

```diff
@@ -56,7 +56,7 @@ In OpenCTI, the different roles are highly customizable. It is possible to defin
 
 ## Presentation of a typical page in OpenCTI
 
-Although there are many different entities in OpenCTI and many different tabs, most of them are quite similar and only have minor differences from the other, mostly due to some of their characteristics, which requires specific fields or do not require some fields which are necessary for the other. 
+Although there are many entities in OpenCTI and many tabs, most of them are quite similar and only have minor differences from the other, mostly due to some of their characteristics, which requires specific fields or do not require some fields which are necessary for the other. 
```

Comment:
A bit long for a single sentence, maybe need to split

Reply:
Done